hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/resolver/backend/SqsBackendResolver.java

```diff
@@ -33,6 +42,17 @@
     * */
     Entity.Builder entityBuilder =
         getBackendEntityBuilder(BackendType.SQS, backendURI.get(), event);
-    return Optional.of(entityBuilder.build());
+    setAttributeForFirstExistingKey(
+        event,
+        entityBuilder,
+        MessagingSemanticConventionUtils.getAttributeKeysForMessagingOperation());
+
+    String attributeKey =
```

Comment:
Move this to method,

```
DbSemanticConventionUtils.getOperation(Event event) {
String attributeKey =
        SpanAttributeUtils.getFirstAvailableStringAttribute(
            event, DbSemanticConventionUtils.getAttributeKeysForDBOperation());
    AttributeValue operation = SpanAttributeUtils.getAttributeValue(event, attributeKey);
}
```
So, for all the db places - redis, mongo, rabbit, etc we can use the same method.

Reply:
Same for messaging related operation code which is there in kafka + sqs -> `MessagingSemanticConventionUtils.getOperation`

Reply:
Sure @rish691 . Shall take care of it in separate PR.